src/test/java/com/nike/cerberus/archaius/client/provider/CerberusConfigurationSourceTest.java

```diff
@@ -94,13 +96,22 @@ public void poll_only_loads_data_for_path1() {
         when(vaultClient.read(PATH_2)).thenThrow(new VaultServerException(500, Lists.newArrayList("Internal error.")));
 
         // call the method under test
-        PollResult result = subject.poll(true, null);
+        subject.poll(true, null);
+    }
 
```

Comment:
Bug fixed cause code coverage to go down slightly so coveralls complained at me.  Adding below tests to satisfy coveralls.